test/RelayHub.test.ts

```diff
@@ -422,6 +422,17 @@ contract('RelayHub', function ([_, relayOwner, relayManager, relayWorker, sender
           )
         })
 
+        it('should revert if encoded function contains extra bytes', async () => {
```

Comment:
please add test for signature require

Reply:
there is, but it the Validator test. here is just a single test, just to verify the "verify" method is called.

Reply:
there is no check for a `require('invalid signature length'`